pkg/collect/cluster_resources.go

```diff
@@ -414,7 +414,16 @@ func getPodDisruptionBudgets(ctx context.Context, client *kubernetes.Clientset,
 	errorsByNamespace := make(map[string]string)
 
 	for _, namespace := range namespaces {
-		PodDisruptionBudgets, err := client.PolicyV1beta1().PodDisruptionBudgets(namespace).List(ctx, metav1.ListOptions{})
+		PodDisruptionBudgets, err := client.PolicyV1().PodDisruptionBudgets(namespace).List(ctx, metav1.ListOptions{})
```

Comment:
I believe this should be like so

```go
		ok, err := discovery.HasResource(client, "policy/v1", "PodDisruptionBudgets")
		if err != nil {
			return nil, map[string]string{"": err.Error()}
		}
		if ok {
			PodDisruptionBudgets, err = client.PolicyV1().PodDisruptionBudgets(namespace).List(ctx, metav1.ListOptions{})
		} else {
			PodDisruptionBudgets, err = client.PolicyV1beta1().PodDisruptionBudgets(namespace).List(ctx, metav1.ListOptions{})
		}
```

Reply:
I see I pushed an erroneous/insufficient version of cluster_resources.go @banjoh fixing this atm.